### PR TITLE
added internationalization

### PR DESCRIPTION
**Pull request by Ksenaaa**

### CODE REVIEW

File: src/components/Header/Contacts/Contacts.tsx [hunk 1]
```diff
@@ -1,22 +1,40 @@
...
 import styles from './Contacts.module.scss'
 
 export const Contacts = () => {
+    const { isOpen: isOpenModalLanguage, onToggle: onToggleModalLanguage } = useToggle()
+
+    const { t, i18n } = useTranslation()
+
+    const lngs = ['en', 'ua']
+
+    const onToggleModal = useCallback(() =>
```
Comment (vell28) [inline src/components/Header/Contacts/Contacts.tsx]: why do we need this function if we have the same function inside?

File: src/components/ModalWindow/ModalWindow.module.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+.modalList {
+    padding: 8px 16px 16px;
+    background: $grey;
+    border-radius: 0px 0px 5px 5px;
+    position: absolute;
+    left: 0;
+    top: 100%;
+    z-index: 999;
+    animation-name: active;
+    animation-duration: 0.4s;
```
Comment (vell28) [inline src/components/ModalWindow/ModalWindow.module.scss]: as i remember we have variable for `animation-duration`?

File: src/components/ModalWindow/ModalWindow.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    icon: boolean
+}
+
+export const ModalWindow: FC<Props> = ({ modalList, isActive, onChangeItem, onToggleModal, icon }) => {
+    const onClickItem = useCallback((itemModal: string) => {
+        onChangeItem(itemModal)
+        onToggleModal()
+    }, [])
+
+    const onCloseModal = () => onToggleModal()
```
Comment (vell28) [inline src/components/ModalWindow/ModalWindow.tsx]: only pas `onToggleModal` as `onClose`. STOP create meaningless function which do nothing

File: src/components/ModalWindow/ModalWindowItem.tsx [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    onClickItem: (item: string) => void,
+    icon: boolean
+}
+
+export const ModalWindowItem: FC<Props> = ({ item, isActive, onClickItem, icon }) => {
+    const onClick = () => onClickItem(item)
+
+    return (
+        <li
+            className={cn(styles.modalItem, isActive === item && styles.isActive)}
```
Comment (vell28) [inline src/components/ModalWindow/ModalWindowItem.tsx]: as a variant you can write `cn(styles.modalItem, { [styles.isActive]: isActive === item })`

File: src/components/Slider/Slider.tsx [hunk 5]
```diff
@@ -35,17 +35,15 @@ export const Slider = () => {
     useEffect(() => {
         if (!sliderPictures?.length) return
 
+        switchingImg(indexPicture)
+
         const timeNext = setTimeout(() => setIndexPicture(preState =>
             sliderPictures.length - 1 === preState ? 0 : preState + 1)
         , 3000)
```
Comment (vell28) [inline src/components/Slider/Slider.tsx]: move duration `3000` to constant

File: src/components/Slider/Slider.tsx [hunk 6]
```diff
@@ -35,17 +35,15 @@ export const Slider = () => {
     useEffect(() => {
         if (!sliderPictures?.length) return
 
+        switchingImg(indexPicture)
+
         const timeNext = setTimeout(() => setIndexPicture(preState =>
```
Comment (vell28) [inline src/components/Slider/Slider.tsx]: setTimeout return as id, would be better change variable's name to `timeoutId`

File: src/components/Slider/SliderItem/SliderItem.tsx [hunk 7]
```diff
@@ -11,15 +14,17 @@ type Props = {
 }
 
 export const SliderItem: FC<Props> = ({ picture }) => {
+    const { t, i18n } = useTranslation()
+
     return (
-        <div className={styles.wrapper}>
+        <div className={cn(styles.wrapper, i18n.language === 'ua' && styles.wrapperUa)}>
```
Comment (vell28) [inline src/components/Slider/SliderItem/SliderItem.tsx]: 'ua' - magic variable,  move it to `enum` and use it

**Review by vell28 — APPROVED** [review]

File: src/components/Header/Contacts/Contacts.tsx (hunk 1 above)
Comment (Ksenaaa) [reply src/components/Header/Contacts/Contacts.tsx]: fixed

File: src/components/ModalWindow/ModalWindow.module.scss (hunk 2 above)
Comment (Ksenaaa) [reply src/components/ModalWindow/ModalWindow.module.scss]: was for transition, added for animation-duration

File: src/components/ModalWindow/ModalWindow.tsx (hunk 3 above)
Comment (Ksenaaa) [reply src/components/ModalWindow/ModalWindow.tsx]: fixed

File: src/components/ModalWindow/ModalWindowItem.tsx (hunk 4 above)
Comment (Ksenaaa) [reply src/components/ModalWindow/ModalWindowItem.tsx]: fixed

File: src/components/Slider/Slider.tsx (hunk 5 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

File: src/components/Slider/Slider.tsx (hunk 6 above)
Comment (Ksenaaa) [reply src/components/Slider/Slider.tsx]: fixed

File: src/components/Slider/SliderItem/SliderItem.tsx (hunk 7 above)
Comment (Ksenaaa) [reply src/components/Slider/SliderItem/SliderItem.tsx]: added to constants

File: src/utils/constants/moveDuration.ts [hunk 8]
```diff
@@ -0,0 +1 @@
+export const moveDuration = 3000
```
Comment (vell28) [inline src/utils/constants/moveDuration.ts]: what is `moveDuration`,  create more understandable variable name

**Review by vell28 — APPROVED** [review]

**Review by valentinPortside — APPROVED** [review]

**Pull request merged**